src/main/java/rearth/oritech/Oritech.java

```diff
@@ -19,6 +19,7 @@
 import rearth.oritech.client.init.ModScreens;
 import rearth.oritech.client.init.ParticleContent;
 import rearth.oritech.init.*;
+import rearth.oritech.init.OritechConfig;
```

Comment:
```suggestion
import rearth.oritech.init.OritechConfig;
```
I believed it was unnecessary, but my stubborn IDE insisted on this.

Reply:
Yeah most IDEs don't seem happy with owo config